src/main/java/io/github/eocqrs/kafka/xml/NameInCamelCase.java

```diff
@@ -23,10 +23,15 @@
 package io.github.eocqrs.kafka.xml;
 
 import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.cactoos.Text;
+import org.cactoos.func.FuncOf;
+import org.cactoos.iterable.IterableOf;
 import org.cactoos.list.ListOf;
-import org.cactoos.text.Concatenated;
-import org.cactoos.text.Joined;
-import org.cactoos.text.TextOf;
+import org.cactoos.proc.ForEach;
+import org.cactoos.text.*;
```

Comment:
don't use *, please

Reply:
@h1alexbel I agree, my IDEA is annoyed by this import section